src/dmd/clone.d

```diff
@@ -315,7 +315,7 @@ FuncDeclaration buildOpAssign(StructDeclaration sd, Scope* sc)
         auto idswap = Identifier.generateId("__swap");
         auto swap = new VarDeclaration(loc, sd.type, idswap, new VoidInitializer(loc));
         swap.storage_class |= STC.nodtor | STC.temp | STC.ctfe;
-        if (tdtor.isscope)
+        if (tdtor.isthisscope)
```

Comment:
Was the old name misleading?

Reply:
See https://github.com/dlang/dmd/pull/11461#discussion_r460561309